### PR TITLE
Feature/renew kickoff

### PR DESCRIPTION
**Pull request by Andrer757**
Updated the dependencies and the template to kotlin

### CODE REVIEW

**Review by extmkv — APPROVED** [review]

File: example/local/project_configurations.json [hunk 1]
```diff
@@ -1,92 +1,225 @@
...
+                },
+                "logging": {
+                    "group": "com.squareup.okhttp3:logging-interceptor",
+                    "version": "4.1.0",
+                    "isCompiler": false
+                }
+            }
+        },
+        {
+            "name": "threeTenABP",
```
Comment (jzeferino) [inline example/local/project_configurations.json:195]: not sure if this should be part of the out of the box version.

File: templates/kotlin/mobile/app/src/main/kotlin/app_package/utils/biometrics/BiometricManagerV23.kt.ftl [hunk 2]
```diff
@@ -0,0 +1,175 @@
+package ${configs.packageName}.utils.biometrics
+
```
Comment (jzeferino) [inline templates/kotlin/mobile/app/src/main/kotlin/app_package/utils/biometrics/BiometricManagerV23.kt.ftl:2]: were does this class come from?

File: example/local/project_configurations.json (hunk 1 above)
Comment (Andrer757) [reply example/local/project_configurations.json:195]: it's just an example

File: templates/kotlin/mobile/app/src/main/kotlin/app_package/utils/biometrics/BiometricManagerV23.kt.ftl (hunk 2 above)
Comment (Andrer757) [reply templates/kotlin/mobile/app/src/main/kotlin/app_package/utils/biometrics/BiometricManagerV23.kt.ftl:2]: this package it's for when you need biometrics authentication in the app.

**Review by jzeferino — APPROVED** [review]

**Pull request merged**